public/js/admin.js

```diff
@@ -1,33 +1,26 @@
+"use strict";
+/* global define, app, socket, bootbox */
+
 define('admin/plugins/markdown', ['settings'], function(Settings) {
 	var Markdown = {};
-	
+
 	Markdown.init = function() {
 		Settings.load('markdown', $('.markdown-settings'), function(err, settings) {
-			var defaults = {
-				'html': false,
-				'xhtmlOut': true,
-				'breaks': true,
-				'langPrefix': 'language-',
-				'linkify': true,
-				'externalBlank': false,
-				'nofollow': true,
-				'typographer': false,
-				'highlight': true,
-				'highlightTheme': 'railscasts.css'
-			};
-	
 			// Set defaults
-			for(var setting in defaults) {
-				if (!settings.hasOwnProperty(setting)) {
-					if (typeof defaults[setting] === 'boolean') {
-						$('#' + setting).prop('checked', defaults[setting]);
-					} else {
-						$('#' + setting).val(defaults[setting]);
+			$.get('/api' + window.location.pathname, function(data) {
```

Comment:
Hm, there shouldn't be any reason to call the api route here. `Settings.load` should populate the form automatically, no?

It looks like the code here _before_ was populating fields that weren't set in the database, via the configured defaults. In this case, since defaults are returned in the API call now, I'd recommend you update this code to reference `ajaxify.data` instead.